src/components/fields/image-upload/image-review/image-review.tsx

```diff
@@ -0,0 +1,423 @@
+import { Icon } from "@lifesg/react-design-system/icon";
+import { Modal } from "@lifesg/react-design-system/modal";
+import { lazy, Suspense, useContext, useEffect, useRef, useState } from "react";
+import IconEraserBlack from "../../../../assets/img/icons/eraser-black.svg";
+import IconEraserBlue from "../../../../assets/img/icons/eraser-blue.svg";
+import IconDeleteDisabled from "../../../../assets/img/icons/image-delete-disabled.svg";
+import IconDelete from "../../../../assets/img/icons/image-delete.svg";
+import IconDrawDisabled from "../../../../assets/img/icons/image-draw-disabled.svg";
+import IconDraw from "../../../../assets/img/icons/image-draw.svg";
+import IconPencilGrey from "../../../../assets/img/icons/pencil-grey.svg";
+import IconPencilWhite from "../../../../assets/img/icons/pencil-white.svg";
+import { FileHelper, ImageHelper, TestHelper } from "../../../../utils";
+import { useFieldEvent, usePrevious } from "../../../../utils/hooks";
+import { TFileCapture } from "../../../shared";
+import { ImageContext } from "../image-context";
+import { ImageUploadHelper } from "../image-upload-helper";
+import { EImageStatus, IImage, TImageUploadAcceptedFileType } from "../types";
+import { IImageEditorRef } from "./image-editor";
+import { ImageError } from "./image-error";
+import { ImagePrompts } from "./image-prompts";
+import {
+	ButtonIcon,
+	ContentSection,
+	DeleteIcon,
+	DrawDeleteButton,
+	DrawDeleteButtonText,
+	DrawDeleteButtonWrapper,
+	DrawIcon,
+	EditHeaderButton,
+	EraserButton,
+	FooterSaveButton,
+	FooterSection,
+	HeaderSection,
+	ImageEditorWrapper,
+	LoadingPreviewText,
+	ModalBox,
+	Palette,
+	PaletteHolder,
+	ReviewCloseButton,
+	ReviewTitle,
+} from "./image-review.styles";
+import { ImageThumbnails } from "./image-thumbnails";
+
+// lazy load to fix next.js SSR errors
+const ImageEditor = lazy(() => import("./image-editor"));
+
+const PALETTE_COLORS = [
+	{ color: "#282828", label: "black" },
+	{ color: "#FFF", colorScheme: "light", label: "white" },
+	{ color: "#017cd3", label: "blue" },
+	{ color: "#fb0101", label: "red" },
+	{ color: "#22910c", label: "green" },
+	{ color: "#f8e821", label: "yellow" },
+];
+interface IProps {
+	accepts: TImageUploadAcceptedFileType[];
+	capture?: TFileCapture;
+	compress?: boolean | undefined;
+	dimensions: { width: number; height: number };
+	id?: string | undefined;
+	maxFiles: number;
+	maxSizeInKb: number;
+	onExit: () => void;
+	outputType: string;
+	show: boolean;
+}
+
```

Comment:
image review modal that will render the uploaded images, image editor and allow more uploads